### PR TITLE
My-React-redux-bookstore-app Milestone-1

### PR DESCRIPTION
**Pull request by Davidosky007**
# My-React-redux-bookstore-app Milestone-1
In this milestone i
* Used create-react-app to initialize a new React app with the --use-npm flag
* Deleted any unnecessary file or asset
* Added prop-types using npm for typechecking props
* Added react-redux and redux using npm
*  deployed the app to Heroku

### CODE REVIEW

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @Davidosky007**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [ ] Your readme file contains some placeholder text. Please make sure to remove it.


---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Comment by Davidosky007** [comment]
> ### Changes required ♻️
> **Hello @Davidosky007**
> 
> Great job with the project so far it looks amazing, but there are still some changes to be made:
> 
> ### Requested 🔴
> * [ ]  Your readme file contains some placeholder text. Please make sure to remove it.
> 
> please make these changes and request another code review.
> 
> if you have any questions don't hesitate to contact me on slack **M

File: README.md [hunk 1]
```diff
@@ -1,70 +1,79 @@
...
-You will also see any lint errors in the console.
+[Live Demo Link](https://bukstur-app.herokuapp.com/)
 
-### `npm test`
+## Getting Started
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```
Comment (mahmoud717) [inline README.md]: this is also template text, make sure to remove it

**Review by mahmoud717 — APPROVED** [review]
### STATUS: APPROVED ✔️ ✔️ 

**hi @Davidosky007**

**Great job with making all of the requirements. the project is approved, you can go ahead and merge this pull request**

File: README.md (hunk 1 above)
Comment (Davidosky007) [reply README.md]: alright thanks 👍

**Pull request merged**